pkg/apis/componentconfig/types.go

```diff
@@ -77,6 +77,9 @@ type ControllerManagerConfiguration struct {
 	OSBAPIContextProfile   bool
 	OSBAPIPreferredVersion string
 
+	// OSBAPITimeOut is the length of the timeout of any request to the broker, in seconds.
+	OSBAPITimeOut int
```

Comment:
Make this a time.Duration like the other fields?

Reply:
OK, this is reasonable. I will recheck it. Thanks.